src/detectors/ScanningStrategyDetector.ts

```diff
@@ -152,6 +152,7 @@ export class ScanningStrategyDetector implements IDetector<string, ScanningStrat
         }
       } catch (error) {
         this.d(error.message);
+        console.log(error.message);
```

Comment:
Is it necessary or you just forgot it here?

Reply:
```suggestion
```